frontend/uddhar-frontend/src/public/components/VOM.jsx

```
@@ -0,0 +1,69 @@
+          <Slider {...settings}>
+            {topVolunteer.map((volunteer) => {
+              const { id, name, location, image } = volunteer;
+              return (
+                <div
+                  key={id}
+                  className="w-auto pb-10 mx-6 my-1 flex flex-col items-center rounded-t-full"
+                >
+                  <img
+                    className="rounded-full h-[300px]"
+                    src={image}
+                    alt={`${name}'s image`}
```

Biome (1.9.4)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)